### DISCUSSION TITLE
prune --keep-from "specific-date-time | archive name"

### DISCUSSION THREAD

**Discussion by tomaszdrozdz** [borgbackup/borg#5752] [Ideas]
When pruning repository, it would be very desired to have possibility to keep all archives "equal" or younger then provided specific "date-time" and delete every older.

Secondly
it would be also great one could provide archive name and the time stamp from it would be retrieved and all "equal" and younger archives then this timestamp would be preserved while older would be erased.

I believe it would not be hard after all, while it would improve  overall Borg usage experience.

**Comment by ThomasWaldmann** [comment]
Well, if you just need that once, guess you could first use borg list --prefix / --glob-archives and then just cut that list as desired and feed it to borg delete.

If you need that more frequently, I'ld be interested in more details about your use case.
